src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -1,22 +1,22 @@
-﻿{
+{
   "$schema": "http://json.schemastore.org/template",
   "author": "Uno Platform",
   "classifications": [
     "Multi-platform",
     "Uno Platform",
     "Android",
     "iOS",
-    "Windows",
     "macOS",
+    "Mac Catalyst",
+    "Windows",
     "Linux",
-    "WebAssembly",
-    "WinUI",
-    "WindowsAppSdk",
-    "Extensions",
-    "dotnet-new"
+    "WebAssembly"
+    // "WinUI",
```

Comment:
We need to keep WinUI, since this is used by both WinAppSdk and TemplateStudio - we want uno templates to show up in this category

Reply:
good catch that wasn't actually supposed to be one that I meant to remove